pkg/cmd/util/output/backup_describer_test.go

```diff
@@ -124,9 +124,10 @@ Or label selector:  <none>
 
 Storage Location:  backup-location
 
-Velero-Native Snapshot PVs:  auto
-Snapshot Move Data:          auto
-Data Mover:                  mover
+Velero-Native Snapshot PVs:    auto
+Default Volumes to Fs Backup:  auto
```

Comment:
1.  I think we need better wording to this field rather than "Default Volumes to Fs Backup". cc @Lyndon-Li 
2. I don't think "auto" is a valid value of this field, it has to be either "true" or "false"

Reply:
1. How about `Pod Volume Backup (Default):` or `File System Backup (Default)` ?
2. How about `"<Not Set>"`

Reply:
@shubham-pampattiwar It will only ever be "not set" before the backup controller reconciles it. If it's not set, then the default value from the server gets copied over. But that is a valid use case for describe, since a backup will be in that state for a while if another backup is currently running when it's created.

Reply:
I am somehow confused with the purpose:
1. If we want to reflect the backup method, we don't need this field at all, because as @sseago mentioned, we don't know the backup method until the backup controller reconciles it; while, we already have this info in the volume info section.
2. If we want to show backup options selected by users, it is enough to make a straight as:
Showing "Default to fs-backup" when `default-to-fs-backup` is true
Showing nothing otherwise

Reply:
@shubham-pampattiwar I mean which is the purpose for this PR, 1 or 2?
If the purpose is 1, I don't think we need to add this change.
If the purpose is 2, we just need to show it when `default-to-fs-backup` is set.

Reply:
@Lyndon-Li the purpose is option 2

Reply:
@shubham-pampattiwar I'm not sure "auto" makes sense here. We use that for includeClusterResources since "auto" describes the behavior -- it's an intermediate state between "include all" and "exclude all" -- "include relevant ones only". But for defaultVolumesToFsBackup, nil just means "the controller hasn't filled in the server default value yet", which is a completely different meaning. Maybe "server default" instead of "auto" here? (in any case, once it's reconciled, it will be filled in with true or false)

Reply:
Updated the PR to only show it when default-to-fs-backup is set.